<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApplication</class>
 <widget class="QWidget" name="MainApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plants vs. Zombies</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/newwindowicon.png</normaloff>:/resource/newwindowicon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="start_page">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resource/openbackground.png);</string>
      </property>
      <widget class="QPushButton" name="click_to_start_button">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>550</y>
         <width>180</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>20</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>180</width>
         <height>20</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton 

{
	border: none;
	/*border-radius:33px;  /*圆形按钮，QPushButton的半径*/
	border-image: url(:/resource/click to start0.png);
	
}
 
QPushButton:hover
{
	border: none;
	/*border-radius:33px;  /*圆形按钮，QPushButton的半径*/
	border-image: url(:/resource/click to start2.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>180</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="menu_page">
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/resource/newnew_menue_page_background.png);</string>
      </property>
      <widget class="QPushButton" name="pushButton_quit">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>510</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton 

{
	border: none;
	
	
	border-image: url(:/resource/quit0.png);
	
}
 
QPushButton:hover
{
	border: none;
	
	
	border-image: url(:/resource/quit1.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_help">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>530</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton 

{
	border: none;
	
	
	border-image: url(:/resource/help0.png);
	
}
 
QPushButton:hover
{
	border: none;
	
	
	border-image: url(:/resource/help1.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_options">
       <property name="geometry">
        <rect>
         <x>567</x>
         <y>490</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton 

{
	border: none;
	
	
	
	border-image: url(:/resource/options8.png);
	
}
 
QPushButton:hover
{
	border: none;
	
	
	
	border-image: url(:/resource/options9.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_achievements">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>499</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton 

{
	border: none;
	
	
	border-image: url(:/resource/achievements0.png);
	
}
 
QPushButton:hover
{
	border: none;
	
	border-image: url(:/resource/achievements1.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_adventure">
       <property name="geometry">
        <rect>
         <x>403</x>
         <y>66</y>
         <width>330</width>
         <height>145</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>145</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton 

{
	border: none;
	
	
	
	border-image: url(:/resource/adventure2.png);
	
}
 
QPushButton:hover
{
	border: none;
	
	
	
	border-image: url(:/resource/adventure1.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="achievement_page">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resource/achievement_page_background.png);</string>
      </property>
      <widget class="QPushButton" name="pushButton_back">
       <property name="geometry">
        <rect>
         <x>127</x>
         <y>53</y>
         <width>90</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton 

{
	border: none;
	
	
	
	border-image: url(:/resource/back0.png);
	
}
 
QPushButton:hover
{
	border: none;
	
	
	
	border-image: url(:/resource/back1.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRepeatDelay">
        <number>300</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="fight_page">
      <property name="minimumSize">
       <size>
        <width>1400</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1400</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resource/newfight_page.jpg);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>600</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>800</width>
           <height>600</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/resource/newfight_page.jpg);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="quik_page">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resource/quik_page.png);</string>
      </property>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>375</y>
         <width>160</width>
         <height>40</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton 

{
	border: none;
	
	
	
	
	border-image: url(:/resource/cancel0.png);
	
}
 
QPushButton:hover
{
	border: none;
	
	
	
	
	border-image: url(:/resource/cancel1.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_quik">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>375</y>
         <width>160</width>
         <height>40</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton 

{
	border: none;
	
	
	
	
	border-image: url(:/resource/quik0.png);
	
}
 
QPushButton:hover
{
	border: none;
	
	
	
	
	border-image: url(:/resource/quik1.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="help_page">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resource/newhelp_page_background.png);</string>
      </property>
      <widget class="QPushButton" name="pushButton_main_menu">
       <property name="geometry">
        <rect>
         <x>327</x>
         <y>522</y>
         <width>153</width>
         <height>37</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>153</width>
         <height>37</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>153</width>
         <height>37</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton 

{
	border: none;
	
	
	
	
	
	border-image: url(:/resource/mainmenu0.png);
	
}
 
QPushButton:hover
{
	border: none;
	
	
	
	
	
	border-image: url(:/resource/mainmenu1.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
